try.html

```diff
@@ -231,6 +231,10 @@ <h3 id="build"> Build </h3>
 </tbody></table>
 <h3 id="downloads"> Downloads </h3>
 <table class='badge'><tbody>
+  <tr><th data-keywords='sourcegraph' data-doc='sourcegraphDoc'> Sourcegraph (# references other repos): </th>
+  <td><img src='/sourcegraph/github.com/gorilla/mux.svg' alt=''/></td>
+  <td><code>https://img.shields.io/sourcegraph/github.com/gorilla/mux.svg</code></td>
+  </tr>
```

Comment:
Same as the comments in index.html. By the way, you should remove the changes you made in index.html; I generate that file automatically, as hinted at the top of index.html.